content/docs/projects/trust-manager/api-reference.md

```diff
@@ -243,6 +250,56 @@ Target is the target location in all namespaces to sync source data to.
 </table>
 
 
+### `Bundle.spec.target.additionalFormats`
+
+
+AdditionalFormats specifies any additional formats to write to the target
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b><a href="#bundlespectargetadditionalformatsjks">jks</a></b></td>
+        <td>object</td>
+        <td>
+          KeySelector is a reference to a key for some map data object.<br/>
+        </td>
+        <td>false</td>
```

Comment:
self review: this isn't a great description for the field, but it comes directly from the CRD and so changing it will be a pain. I think it's fine, the main documentation file is clear enough IMO